CODEOWNERS

```diff
@@ -26,8 +26,27 @@
 /bgp/ @cilium/sig-bgp
 /cmd/ @cilium/cli
 /clustermesh/ @cilium/sig-clustermesh
-/connectivity/ @cilium/cli
+/connectivity/ @cilium/ci-structure
+/connectivity/check/ipcache.go @cilium/ipcache
+/connectivity/check/metrics*.go @cilium/metrics
+/connectivity/check/policy.go @cilium/sig-policy
+/connectivity/tests/egressgateway.go @cilium/egress-gateway
+/connectivity/tests/encryption.go @cilium/sig-encryption
+/connectivity/tests/errors.go @cilium/sig-agent @cilium/sig-datapath
+/connectivity/tests/externalworkload.go @cilium/sig-clustermesh
+/connectivity/tests/from-cidr.go @cilium/sig-policy
+/connectivity/tests/health.go @cilium/sig-agent
+/connectivity/tests/host.go @cilium/sig-agent
+/connectivity/tests/ipsec_xfrm.go @cilium/ipsec
+/connectivity/tests/perfpod.go @cilium/sig-datapath
+/connectivity/tests/pod.go @cilium/sig-agent
+/connectivity/tests/service.go @cilium/sig-lb
+/connectivity/tests/testloop.sh @jrajahalme
```

Comment:
The script looks simple enough, does it need to be Jarno explicitly?

Reply:
Is it even worth to keep that script around? I doesn't seem to be used by the connectivity tests themselves and according to 15df561e92cc29dd76b18d6be7c3cfc5b299c76b was only used for dev testing.

Reply:
When Michi and I were going through this, we thought it was just a funny easter egg thing :smile:. I'm not against removing it.

cc @jrajahalme

Reply:
😹